src/Calculator.ts

```diff
@@ -144,3 +235,10 @@ export function calculateRaw(
     return results.errorMessage ?? 'something happened with no error message';
   }
 }
+
+function populationDisplay(type: PopulationType): string {
+  // Code->Display https://terminology.hl7.org/1.0.0/CodeSystem-measure-population.html
+  // Always same pattern, example: 'initial-population' -> 'Initial Population'
+  const words: string[] = type.split('-');
+  return words.map(word => word.charAt(0).toUpperCase() + word.slice(1)).join(' ');
+}
```

Comment:
This does always work. It would be more flexible than mapping if any population types were added... but since I don't think that's ever going to happen, I think I like yours better.